src/App.tsx

```diff
@@ -83,11 +72,7 @@ export const App: React.FC = () => {
                 )}
 
                 {author && loaded && !hasError && posts.length > 0 && (
```

Comment:
```suggestion
                {author && loaded && !hasError && !!posts.length && (
```
pls move all long ternaries to separate consts